gum/backend-arm/gumstalker-arm.c

```diff
@@ -3624,6 +3694,10 @@ static void
 gum_exec_block_write_arm_exec_event_code (GumExecBlock * block,
                                           GumGeneratorContext * gc)
 {
+  /* Suppress events during exclusive load/store operations */
+  if (gc->exclusive_load_offset != GUM_INSTRUCTION_OFFSET_NONE)
+    return;
```

Comment:
Would be good to replace this comment with a more descriptive helper function. Perhaps something like:

```C
  if (gum_generator_context_is_timing_sensitive (gc))
    return;
```